apps/condo/domains/common/utils/serverSchema/changeTrackable.test.js

```diff
@@ -0,0 +1,138 @@
+const { generateChangeTrackableFieldsFrom } = require('./changeTrackable')
```

Comment:
Your module public interface has three functions. But you test only one

Reply:
Will be implemented in https://doma.atlassian.net/browse/SBERDOMA-391